src/utils/ucc_parser.c

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (C) Mellanox Technologies Ltd. 2021.  ALL RIGHTS RESERVED.
+ * See file LICENSE for terms.
+ */
+
+#include "ucc_parser.h"
+#include "ucc_malloc.h"
+#include "ucc_log.h"
+
+ucc_status_t ucc_config_names_array_dup(ucc_config_names_array_t *dst,
```

Comment:
why not to use ucs_config_clone_array and ucs_config_release_array?

Reply:
I checked: those routines do different thing: they clone the whole array of parameters (ucs_config_field_t array). Here is just want to create a copy of names array